src/main/resources/templates/clojure-lsp/clientSettings.json

```diff
@@ -1,7 +1,8 @@
 {
   "caseSensitive": true,
   "completion": {
-    "useContextAwareSorting": true
+    "useContextAwareSorting": true,
+    "useTemplateForSingleArgument": false
```

Comment:
Even though editor templates are enabled by default for backward-compatibility, I updated all of the existing language server definition templates to disable editor templates for single-arg invocations because they're so disruptive to the editor typing experience. This is the same approach we used for `useContextAwareSorting` to avoid changing behavior for existing language server definitions while making newly-created ones use the preferred behavior.